java/ql/lib/upgrades/b9225587bc0a643ae484ec215b9a6f19d17d0fc2/upgrade.properties

```diff
@@ -0,0 +1,3 @@
+description: Remove unneeded break/continue structures for Kotlin
+compatibility: backwards
```

Comment:
There should be a `delete` line for the removed relation (see https://github.com/github/codeql/blob/main/docs/prepare-db-upgrade.md)

Reply:
> I also changed the compatibility to `partial`

Why? `backwards` sound right to me.

Reply:
You're probably right. Removing the relation and the corresponding QL classes would not produce worse results than before. The results were already wrong as CFG didn't take into account any of these break/continue targets.